From 5119e37e7e90d85cc0e28a579494394c4709be97 Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Mon, 26 Dec 2011 18:20:11 -0500
Subject: debian: osconf.hin path changes

---
 src/include/osconf.hin | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/include/osconf.hin b/src/include/osconf.hin
index c3a33c2..00c00d6 100644
--- a/src/include/osconf.hin
+++ b/src/include/osconf.hin
@@ -116,8 +116,8 @@
  * krb5 slave support follows
  */
 
-#define KPROP_DEFAULT_FILE "@LOCALSTATEDIR/krb5kdc/slave_datatrans"
-#define KPROPD_DEFAULT_FILE "@LOCALSTATEDIR/krb5kdc/from_master"
+#define KPROP_DEFAULT_FILE "/var/lib/krb5kdc/slave_datatrans"
+#define KPROPD_DEFAULT_FILE "/var/lib/krb5kdc/from_master"
 #define KPROPD_DEFAULT_KDB5_UTIL "@SBINDIR/kdb5_util"
 #define KPROPD_DEFAULT_KPROP "@SBINDIR/kprop"
 #define KPROPD_DEFAULT_KRB_DB DEFAULT_KDB_FILE
